tests/vxlan/test_vxlan_ecmp.py

```diff
@@ -575,10 +575,11 @@ def setUp(duthosts, ptfhost, request, rand_one_dut_hostname, minigraph_facts,
         encap_type = "{}_in_{}".format(payload_version, outer_layer_version)
         set_routes_in_dut(data['duthost'], data[encap_type]['dest_to_nh_map'], payload_version, "DEL")
 
-        for intf in data[encap_type]['vnet_intf_map'].keys():
-            for entry in minigraph_facts['minigraph_interfaces'] + minigraph_facts['minigraph_portchannel_interfaces']:
-                if intf == entry['attachto']:
-                    data['duthost'].shell("sudo config interface ip add {} {}".format(intf, entry['subnet']))
+        for intf in data[encap_type]['selected_interfaces']:
+            redis_string = "INTERFACE"
+            if "PortChannel" in intf > 0:
```

Comment:
I think there's something wrong here.
We're checking `intf > 0`, but intf is, e.g. `'Ethernet240'`, so `intf > 0` results True then `if "PortChannel" in True` doesn't make sense, cause it will always be False.

Reply:
@rraghav-cisco could you please check this on your side

Reply:
Checking.